akka-stream-tests/src/test/scala/akka/stream/io/FileSourceSpec.scala

```diff
@@ -107,6 +121,17 @@ class FileSourceSpec extends StreamSpec(UnboundedMailboxConfig) {
       c.expectComplete()
     }
 
+    "read contents from a named pipe or FIFO" in assertAllStagesStopped {
+      val chunkSize = 512
+      val (future, p) = FileIO.fromPath(testFifo, chunkSize)
+        .addAttributes(Attributes.inputBuffer(1, 2))
+        .toMat(TestSink.probe)(Keep.both).run()
+      p.request(1)
+      p.expectNext().utf8String should ===(TestText.splitAt(chunkSize)._1)
+      p.cancel()
+      Await.result(future, 3.seconds) === createSuccessful(chunkSize)
```

Comment:
The spec class extends `ScalaFutures` (from AkkaSpec, via StreamSpec) so you can just use `.futureValue`. I see other tests are using `Await.result` though; I wonder if there's a reason for using that over `futureValue`? Among other things, `futureValue` will allow scaling with scalatest's patience config etc.

Reply:
I think if the above stands, this particular test is likely not going to be there, but can certainly update the other tests to use `futureValue` and am happy to do so!